hms-app/src/actions/IdeaActions.tsx

```diff
@@ -2,6 +2,20 @@ import { Idea } from '../common/types'
 
 const core_url = process.env.REACT_APP_CORE_URL
 
+export const getIdeaList = (hackathonID: string) => {
+  return fetch(`${core_url}/hackathon/${hackathonID}/ideas`, {
+    method: 'GET',
+    headers: {
```

Comment:
I've looked through the repo and it seems like we have a loot of common fetches. Maybe we could extract the default header set as constant in some `common/request.js` . Same for default actions like map `data => data.json()` or `err => console.log(err)`. We could also create a custom hook, that would handle a call with all the defaults and possibly typed return value. `export const getIdeaList = useFetch<OurType>('http://hackaton/xyz')` 🤔 Wdyt? 😄

Reply:
I second that, would save a lot of lines of code